easybuild/easyconfigs/t/texinfo/texinfo-5.2-GCC-4.8.2.eb

```diff
@@ -16,6 +11,18 @@ sources = [SOURCELOWER_TAR_XZ]
 
 toolchain = {'name': 'GCC', 'version': '4.8.2'}
 
+osdependencies = ['texlive']
+
+preinstallopts = "make TEXMF=%(installdir)s/texmf install-tex && "
+
+# This will overwrite a users TEXMFHOME so this module is best used as a build dependency
+modextravars = {'TEXMFHOME': '%(installdir)s/texmf'}
+modloadmsg    = """\n\nThis module will set/overwrite the value for the environment variable TEXMFHOME."""
+modloadmsg    += """\nIf you use a custom texmf directory (such as ~/texmf) then you should copy files found in the"""
+modloadmsg    += """\nnew TEXMFHOME to your custom directory and reset the value of TEXMFHOME to point to that space:"""
+modloadmsg    += """\n\tcp -r $TEXMFHOME/* /path/to/your/texmf"""
+modloadmsg    += """\n\texport TEXMFHOME=/path/to/your/texmf\n\n"""
+
 sanity_check_paths = {
     'files': ['bin/info', 'bin/makeinfo', 'bin/pod2texi', 'bin/texi2pdf'],
     'dirs': [],
```

Comment:
enhance the sanity check too, to check for (some of) the stuff that was missing but should now be there?